<problem chapter='2' section='6' slide='1' problem='7' title="Hibák – a programozók mindennapi kenyere | try-except">

<!--
Update history:
2020-09-21 Grzegorz (review, PEP-8 compliance check, update, title correction)
-->

<code>
try:
    print("1")
    x = 1 / 0
    print("2")
except:
    print("Oh dear, something went wrong...")

print("3")

</code>



<h1>Kivételek (folytatás)</h1>

<p>Nézd meg a kódot a szerkesztõben. Ez segíteni fog megérteni ezt a mechanizmust.</p>

<p>Ezt a kimenetet produkálja:</p>

<codep>1
Oh dear, something went wrong...
3</codep>
<p style="text-align: right;"><font size="-2"><b>kimenet</b></font></p>


<p>Megjegyzés: a <codel>print("2")</codel> utasítás elveszett a folyamat során.</p>


<br>
<br>

<studyMode>false</studyMode>
<labMode>false</labMode>

</problem>
